src/Mount.cpp

```diff
@@ -2199,7 +2199,7 @@ void Mount::startSlewing(int direction) {
       #endif
 
       if (direction & NORTH) {
-        long targetLocation = sign * 300000;
+        long targetLocation = 300000;
```

Comment:
Hmm, ok. I guess if that's not needed for the Southern hemisphere, then you can delete line 2177.,

Reply:
I think it's still needed for RA, so that left/right corresponds to CW/CCW (well this is what I read in docs as expected behaviour).

Reply:
Forgot to mention, that's in lines 2231 & 2236

Reply:
Oh boy, i swear i searched for it and found no other uses....